source/Service.js

```diff
@@ -445,9 +473,14 @@ class Service extends EventEmitter {
                 direction: order
             }
         ]);
+        //await this.logger.addEntry(this.logger.levels.LOGGER_DEBUG, "Testing the logger");
         return limit !== Infinity ? jobs.slice(0, limit) : jobs;
     }
 
+    readLog() {
```

Comment:
Not sure if we need a helper method here, calling `.logger` is nice and detached :)

Reply:
Done, changed the UI's routes to call the logger directly also :)